server/migrations/2021-04-03-161950_add_dates/up.sql

```diff
@@ -0,0 +1,33 @@
+ALTER TABLE users
+    ADD created timestamp NOT NULL default current_timestamp,
+    ADD updated timestamp NOT NULL default current_timestamp;
+
+ALTER TABLE rooms
+    ADD created timestamp NOT NULL default current_timestamp,
+    ADD updated timestamp NOT NULL default current_timestamp;
+
+ALTER TABLE rooms_users
+    ADD created timestamp NOT NULL default current_timestamp,
+    ADD updated timestamp NOT NULL default current_timestamp;
+
+CREATE FUNCTION last_upd_trig() RETURNS trigger
+    LANGUAGE plpgsql AS
+$$BEGIN
+    NEW.updated := current_timestamp;
```

Comment:
This should fix the issue with the database, as `current_timestamp` does not change during a transaction, whereas `clock_timestamp` does.

```suggestion
    NEW.updated := CLOCK_TIMESTAMP();
```